setup.py

```diff
@@ -24,7 +24,11 @@
         'python-dateutil',
         'pytz',
         'webob',
+        'fs',
     ],
+    extras_require={
+        'django-fs-support': ['django-pyfs']
```

Comment:
A runtime that uses django, and wants the provided django-backed filesystem.

Although, I might just shorten it to `'django': ['django-pyfs']` and plan to include all django-specific dependencies in it.